### ISSUE TITLE
Checkerboard patterns in figure 12 

### ISSUE THREAD

**Issue by balthazarenhance** [erfect2020/TransformerPerceptualLoss#3]
hi, I read your paper and it is quite interesting 
Regarding figure 12, the MAE based perceptual **LOCAL** loss leads to "repetitive patterns".
![image](https://github.com/erfect2020/TransformerPerceptualLoss/assets/167073614/b3732c68-f08f-40f6-9b64-c67b4bbedc35)
I have several questions:
1/ Do you also observe this with VGG loss?
2/ In your opinion, why does this checkerboard pattern disappear with the projected distribution loss ?

Thanks

**Comment by erfect2020** [comment]
After check the source file, I did not find the  "checkerboard pattern' artifacts. I think it is introduced by the compression of PDF.

 I recommend using \( $L_{percep}^{local}$ \) because it is easier to use and less sensitive to hyperparameter. In contrast, VGGloss and \( $L_{percep}^{global}$ \) require careful fine-tuning to balance distortion and perceptual quality.
 
 
![1P0A1527_localv2_cropped](https://github.com/erfect2020/TransformerPerceptualLoss/assets/94505384/39b24efc-9f6e-4727-9b35-c6d23b7069a6)
$L_{percep}^{local}$
